pkg/subctl/cmd/gather.go

```diff
@@ -75,6 +76,8 @@ func addGatherFlags(gatherCmd *cobra.Command) {
 	gatherCmd.Flags().StringVar(&directory, "dir", "",
 		"the directory in which to store files. If not specified, a directory of the form \"submariner-<timestamp>\" "+
 			"is created in the current directory")
+	gatherCmd.Flags().BoolVar(&includeSecrets, "include-secrets", false,
+		"do not redact sensitive data")
```

Comment:
```suggestion
	gatherCmd.Flags().BoolVar(&includeSecrets, "include-sensitive-data", false,
		"do not redact sensitive data such as credentials and security tokens")
```

Reply:
Oh, right, include-secrets sounded good to me, but sensitive-data may be is more precise. 

Anyway, simple & effective implementation thanks @vthapar !! :)